man/systemd.network.xml

```diff
@@ -398,6 +398,18 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>IPv4LLStartAddress=</varname></term>
+        <listitem>
+          <para>Specifies the first IPv4 link-local address to try. Takes an IPv4 address,
+          for example <literal>169.254.1.2</literal> from the link-local address range <literal>169.254.0.0/16</literal>.
```

Comment:
Misplaced comma twists the meaning of this sentence. Please add the comma after "for example 169.254.1.2". Also please drop `<literal>s` — we don't need them around ip address ranges.

Reply:
Why does the "ServerAddress" have the IP address with the `<literal>s`?

```
<varlistentry>
        <term><varname>ServerAddress=</varname></term>
        <listitem><para>Specifies server address for the DHCP server. Takes an IPv4 address with prefix
        length, for example <literal>192.168.0.1/24</literal>. This setting may be useful when the link on
        which the DHCP server is running has multiple static addresses. When unset, one of static addresses
        in the link will be automatically selected. Defaults to unset.</para></listitem>
      </varlistentry>
```

Reply:
Hehe, let's also drop them.

Reply:
I have add an additional commit and drop them from `ServerAddress`